contracts/contracts/strategies/LidoWithdrawalStrategy.sol

```diff
@@ -0,0 +1,307 @@
+        address _recipient,
+        // solhint-disable-next-line no-unused-vars
+        address _asset,
+        // solhint-disable-next-line no-unused-vars
+        uint256 _amount
+    ) external override onlyVault nonReentrant {
+        // Does nothing - all withdrawals need to be called manually using the
+        // Strategist calling claimWithdrawals
+        revert("use claimWithdrawals()");
+    }
+
+    /**
+     * @notice Claim previously requested withdrawals that have now finalized.
+     * Called by the Strategist.
+     * @param _requestIds Array of withdrawal request identifiers
+     * @param expectedAmount Total amount of ETH expect to be withdrawn
+     */
+    function claimWithdrawals(
+        uint256[] memory _requestIds,
+        uint256 expectedAmount
+    ) external nonReentrant {
+        require(
+            msg.sender == IVault(vaultAddress).strategistAddr(),
+            "Caller is not the Strategist"
+        );
+        uint256 startingBalance = payable(address(this)).balance;
+        uint256 lastIndex = withdrawalQueue.getLastCheckpointIndex();
+        uint256[] memory hintIds = withdrawalQueue.findCheckpointHints(
+            _requestIds,
+            1,
+            lastIndex
+        );
+        withdrawalQueue.claimWithdrawals(_requestIds, hintIds);
+
+        uint256 currentBalance = payable(address(this)).balance;
+        uint256 withdrawalAmount = currentBalance - startingBalance;
+        // Withdrawal amount should be within 2 wei of expected amount
+        require(
+            withdrawalAmount + 2 >= expectedAmount &&
+                withdrawalAmount <= expectedAmount,
+            "Withdrawal amount not expected"
+        );
+
+        emit WithdrawalClaims(_requestIds, withdrawalAmount);
+
+        outstandingWithdrawals -= withdrawalAmount;
+        weth.deposit{ value: currentBalance }();
+        // slither-disable-next-line unchecked-transfer
+        weth.transfer(vaultAddress, currentBalance);
+        emit Withdrawal(
+            address(weth),
+            address(withdrawalQueue),
+            currentBalance
+        );
+    }
+
+    function _abstractSetPToken(address, address) internal override {
+        revert("No pTokens are used");
+    }
+
+    /**
+     * @notice Withdraw all assets from this strategy, and transfer to the Vault.
+     * In correct operation, this strategy should never hold any assets.
+     */
+    function withdrawAll() external override onlyVaultOrGovernor nonReentrant {
+        if (payable(address(this)).balance > 0) {
+            weth.deposit{ value: payable(address(this)).balance }();
+        }
+        uint256 wethBalance = weth.balanceOf(address(this));
+        if (wethBalance > 0) {
+            // slither-disable-next-line unchecked-transfer
+            weth.transfer(vaultAddress, wethBalance);
+            emit Withdrawal(address(weth), address(0), wethBalance);
+        }
+        uint256 fraxEthBalance = stETH.balanceOf(address(this));
```

Comment:
Here the name of the function refers to an other liquid staking protocol.

Reply:
good pickup. I did a search and replace for frxEth but missed fraxEth